docs-v2/data/maturity.json

```diff
@@ -88,7 +88,7 @@
     "debug": "x",
     "area": "Build",
     "feature": "ko builder",
-    "maturity": "alpha",
+    "maturity": "beta",
```

Comment:
I hope this is ok :-D. Please let me know if it's not, I'd be happy to revert it.

Reply:
👍